ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -335,6 +340,8 @@ spec:
           subPath: index
         - name: credentials
           workspace: registry-credentials
+        - name: cacert
```

Comment:
Did you intend to add the workspace to `generate-index`? If so, the task also needs to be updated.

Reply:
This PR is only to handle self-signed certificate in buildah task. To run whole pipeline would me one of my next steps

Reply:
It is needed because i am getting following error 
```
failed: [localhost] (item=../templates/openshift/pipelines/operator-ci-pipeline.yml) => {"ansible_loop_var": "item", "changed": false, "error": 400, "item": "../templates/openshift/pipelines/operator-ci-pipeline.yml", "msg": "Pipeline operator-ci-pipeline: Failed to apply object: b'{\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"admission webhook \\\\\"validation.webhook.pipeline.tekton.dev\\\\\" denied the request: validation failed: invalid value: pipeline task \\\\\"build-bundle\\\\\" expects workspace with name \\\\\"registry-cacert\\\\\" but none exists in pipeline spec: spec.tasks[13].workspaces[2]\",\"reason\":\"BadRequest\",\"code\":400}\\n'", "reason": "Bad Request", "status": 400}
```